pymongo/database.py

```diff
@@ -162,7 +162,11 @@ def with_options(
           >>> db1.read_preference
           Primary()
           >>> from pymongo import ReadPreference
-          >>> db2 = db1.with_options(read_preference=ReadPreference.SECONDARY)
+          >>> db2 = db1.with_options(
+                                     read_preference=Secondary(tag_sets=[..]),
```

Comment:
We should put actual values in here for demonstration purposes

Reply:
+1  like:
```
         >>> from pymongo.read_preferences import Secondary
          >>> db2 = db1.with_options(read_preference=Secondary(tag_sets=[{"node":"analytics"}]))
          >>> db1.read_preference
          Primary()
          >>> db2.read_preference
          Secondary(tag_sets=[{'node': 'analytics'}], max_staleness=-1, hedge=None)
```